README.md

```diff
@@ -104,7 +104,6 @@ See also GitHub's [documentation on starter workflows](https://docs.github.com/e
 Executable scripts that are used in our workflows.
 
 - [interpolate-env](bin/interpolate-env)
-- [write-envdir](bin/write-envdir)
```

Comment:
The references to `ENV_DIR` in the workflow (its definition, its use in upload-artifacts) can go away too.

Reply:
Thanks for catching that! Updated in [force-push](https://github.com/nextstrain/.github/compare/1097fc9052d2a07c393a210aa57f1da09b6c1867..822187aa78c560d85be378490d3c666cf21457e5)